spec/features/studio/index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe "the studio index", type: :feature do
+  before(:each) do
+    @studio_1 = Studio.create!(name: "Miramax", location: "LA")
+    @studio_2 = Studio.create!(name: "Disney", location: "Orlando")
+    @studio_1.movies.create!(title: "Jaws", creation_year: 1980, genre: "horror")
+    @studio_1.movies.create!(title: "Top Gun", creation_year: 1987, genre: "action")
+  end
+  # Story 1 - Studio Index
+
+  # And under each studio I see all of the studio's movies
+  # including the movie's title, creation year, and genre
+  it "lists all the studios and their attributes" do
+
+    visit "/studios"
+    save_and_open_page
+    
+    expect(page).to have_content("Miramax")
+    expect(page).to have_content("LA")
```

Comment:
Would be better to test for what this attribute belongs to, something like: 
```
expect(page).to have_content("Location: LA")
```

Reply:
Also, for index pages or any views that have any `.each`s in them, I like to use a `within` block in my feature tests, since the data under `.each` will change any time objects are created or edited.